snapcraft.yaml

```diff
@@ -95,5 +97,6 @@ parts:
       craftctl default
     override-prime: |
       craftctl default
+      rm -f etc/systemd/user/pipewire-session-manager.service
```

Comment:
Can you move this to `hooks/006-add-gdm.chroot`, where we perform similar fixes?